### PR TITLE
updated getting-started with mitigated pitfalls

### PR DESCRIPTION
**Pull request by PatrickAlphaC**
# What :computer: 
1. Updated the recommended faucet -> Chainstack
2. Added a comment about private key use in a .env file 
3. Removed broken atlas link 

# Why :hand:
1. The learnweb3 faucet asks for very invasive GitHub authentication. See attached image below. I have informed the learnweb3 team to fix this. 
2. We teach to always avoid [private keys in .env files](https://www.youtube.com/watch?v=VQe7cIpaE54) as it's one of the most common ways new web3 developers (experienced or not) lose funds. We even have students pledge to never put private keys with funds in them, and it's an anti-pattern that has been propagated across web3 hurting new people. [pledge one](https://github.com/smartcontractkit/full-blockchain-solidity-course-js/discussions/5) and [pledge 2](https://github.com/Cyfrin/foundry-full-course-f23/discussions/5#discussioncomment-8839188)
3. It's not a good idea to have students go to a potentially unmaintained example, see it not work, and then go "ah, f this. This chain just doesn't work." and move to another L2, or worse, out of web3 because of it. I left a comment about how this could be pretty easily fixed with a github actions that points to your templates folder. If you give me access to one I can make it for you and have atlas always paired with an up-to-date example. 

# Images
Screenshot 2024-03-21 at 9 34 11 PM
vidence :camera:

# Notes :memo:

### CODE REVIEW

File: docs/build/quick-start/hello-world.md [hunk 1]
```diff
@@ -31,19 +31,25 @@ This is what we're going to do:
...
+
+I did something similar with Chainlink that is running to this day if you want to see the actions file:
+https://github.com/smartcontractkit/chainlink-brownie-contracts/blob/main/.github/workflows/add-and-commit.yml
+
 ::: info Project available in Atlas IDE
 This entire tutorial can be run in under a minute using Atlas. Atlas is a smart contract IDE that lets you write, deploy, and interact with contracts from your browser.
 
-[Open this project in Atlas](https://app.atlaszk.com/projects?template=https://github.com/matter-labs/zksync-hardhat-template&open=Greeter.sol&chainId=300).
-:::
+[Open this project in Atlas](https://app.atlaszk.com/projects?template=https://github.com/matter-labs/zksync-contract-templates&open=Greeter.sol&chainId=300).
```
Comment (PatrickAlphaC) [inline docs/build/quick-start/hello-world.md:51]:
```suggestion
[Open this project in Atlas](https://app.atlaszk.com/projects?template=https://github.com/matter-labs/zksync-hardhat-template&open=Greeter.sol&chainId=300).
```

**Comment by MexicanAce** [comment]
For awareness, I've tried to validate the new repo would work with Atlas, but I'm getting `Unable to Create Project: unhashable type: 'slice'` error:

* https://app.atlaszk.com/projects?template=https://github.com/matter-labs/zksync-contract-templates/tree/main/templates/hardhat/solidity&open=Greeter.sol&chainId=300

I can follow-up with Atlas to troubleshoot and resolve

File: docs/build/quick-start/hello-world.md [hunk 2]
```diff
@@ -63,7 +69,7 @@ It will give you options for different types of projects but for this tutorial c
 
 ::: info
 
-The private key of your wallet will be included in the `.env` file of the project and won't be pushed to GitHub.
+The private key of your wallet will be included in the `.env` file of the project and won't be pushed to GitHub. However, as this means the private key is in plain text, it's recommended to use a private key that isn't associated with any real funds.
```
Comment (MexicanAce) [inline docs/build/quick-start/hello-world.md:72]: I'm fine with this single change for now, but we should update all tutorials that use this same `::: info` tag with this additional warning.

Comment (MexicanAce) [reply docs/build/quick-start/hello-world.md:72]: Issue created: https://github.com/matter-labs/zksync-web-era-docs/issues/985

File: docs/build/quick-start/hello-world.md [hunk 3]
```diff
@@ -31,19 +31,25 @@ This is what we're going to do:
...
-You can get testnet ETH directly into zkSync testnet using the [following faucet provided by LearnWeb3](https://learnweb3.io/faucets/zksync_sepolia/).
+You can get testnet ETH directly into zkSync testnet using the [following faucet provided by Chainstack](https://faucet.chainstack.com/zksync-testnet-faucet).
 
 Another option is to [get SepoliaETH from any of the following faucets](../tooling/network-faucets.md) and bridge it to zkSync Sepolia testnet using the [zkSync bridge](https://portal.zksync.io/bridge/?network=sepolia).
 
 You can check the balance of your account in the [zkSync Sepolia explorer](https://sepolia.explorer.zksync.io/).
 
 ## Create the project
 
+<!--
```
Comment (MexicanAce) [inline docs/build/quick-start/hello-world.md:42]: Please run `yarn fix:fmt` to resolve prettier linting issues with this branch

**Review by MexicanAce — APPROVED** [review]
LGTM, created open issue to close conversation and will follow-up in separate PR when received a fix from Atlas for new repo

**Pull request merged**